ChangeLog.d/psa-crypto-key-policies.txt

```diff
@@ -0,0 +1,5 @@
+Features
+   * A key in PSA Crypto can now be defined with a policy instead of a specific
+     algorithm. Policies describe a collection of allowed specific algorithms.
```

Comment:
The generic possibility is not new: it already exists for signature keys that can be declared as permitted with any hash.